vite.config.ts

```diff
@@ -23,4 +23,15 @@ export default defineConfig(({ mode }) => ({
         sourcemap: mode == 'dev',
         emptyOutDir: true,
         copyPublicDir: false,
-    }}));
+    },
+    server: {
+        proxy: {
+            '/api': {
+                target: 'https://scicrunch.org',
+                changeOrigin: true,
+                rewrite: (path) => path.replace(/^\/api/, '/api/1/elastic/Interlex_pr/_search?key=I0FJ8VwuqLSVxfW586ICMpgzdT1m7miU'),
```

Comment:
critical: @afonsobspinto as mentioned today on slack, it's not right to put "keys", "passwords" etc in Git